tools/lkl/lib/virtio_net_dpdk.c

```diff
@@ -33,9 +33,14 @@ static char *ealargs[4] = {
 /* XXX: disable cache due to no thread-safe on mempool cache. */
 #define MEMPOOL_CACHE_SZ        0
 /* for TSO pkt */
-#define MAX_PACKET_SZ           (65535 \
+//For ixgbe and vmxnet3 drivers
+/*#define MAX_PACKET_SZ           (65535 \*/
+/*	- (sizeof(struct rte_mbuf) + RTE_PKTMBUF_HEADROOM))*/
+//For Intel i350 NIC (igb driver)
+#define MAX_PACKET_SZ           (16383 \
 	- (sizeof(struct rte_mbuf) + RTE_PKTMBUF_HEADROOM))
-#define MBUF_NUM                (512*2) /* vmxnet3 requires 1024 */
+//#define MBUF_NUM                (512*2) /* vmxnet3 requires 1024 */
+#define MBUF_NUM                4096 /* Intel i350 NIC (igb driver): 4096 */
```

Comment:
Could we set these macros to the minimum to have all boards working, and then add some new ifparams like num_bufs and max_packet_size? The the user can optimize with LKL_HIJACK_NET_IFPARAMS="num_bufs=..." for a particular nic.

Or we could add a new ifparam (e.g. nic=ixgbe) that we can parse and set the above as a convenience for the user...

I don't fill too strong about these, lets see what @thehajime thinks.